item.rb

```diff
@@ -0,0 +1,38 @@
+require 'date'
+
+class Item
+  attr_reader :label, :genre, :author, :source, :publish_date, :archived, :id
+
+  def initialize(id, publish_date)
+    @id = id
+    @publish_date = publish_date
+    @archived = false
+  end
+
+  def add_genre(genre)
+    @genre = genre.name
+    genre.items.push(self) unless genre.items.include?(self)
+  end
+
+  def add_author(author)
+    @author = author.first_name
+    author.items.push(self) unless author.items.include?(self)
+  end
+
+  def label=(label)
+    @label = label.title
+    label.items << self unless label.items.include?(self)
+  end

```

Comment:
@ichala Sorry for that you are right, it was a mistake by me so please  just send for another review and it will get approved by the reviewer I don't why I didn't see the modified code